### PR TITLE
Fix error of suppress_exception value

### PR DESCRIPTION
**Pull request by suqinhuang**
suppress_exception always be True currently

Signed-off-by: Suqin Huang <shuang@redhat.com>

### CODE REVIEW

**Comment by suqinhuang** [comment]
ID: 1338572

File: qemu/tests/driver_in_use.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def run_subtest(sub_type):
     sub_type = params["sub_test"]
     bg_stress_test = params["run_bgstress"]
     wait_time = float(params.get("wait_bg_time", 60))
-    suppress_exception = bool(params.get("suppress_exception", "no"))
+    suppress_exception = bool(params.get("suppress_exception", "no") == "yes")
```
Comment (CongLi) [inline qemu/tests/driver_in_use.py]: >>> suppress_exception = "no"
>>> bool(suppress_exception)
True

It's better to update suppress_exception to True / False.

Comment (xutian) [inline qemu/tests/driver_in_use.py]: really no need to do it.  suppress_exception = params.get("suppress_exception", "no") == "yes" is enough

File: qemu/tests/cfg/balloon_in_use.cfg [hunk 2]
```diff
@@ -43,22 +43,22 @@
     variants:
         - with_stop_continue:
             sub_test = stop_continue
-            suppress_exception = False
+            suppress_exception = no
         - with_shutdown:
             sub_test = shutdown
             suppress_exception = True
```
Comment (CongLi) [inline qemu/tests/cfg/balloon_in_use.cfg]: Miss this line.

**Comment by CongLi** [comment]
Acked-by: Cong Li.

**Comment by xutian** [comment]
looks good to me

**Pull request merged**